lib.go

```diff
@@ -207,7 +201,7 @@ func (vm *VM) Query(
 	gasMeter GasMeter,
 	gasLimit uint64,
 	deserCost types.UFraction,
-) ([]byte, uint64, error) {
+) (*types.QueryResponse, uint64, error) {
```

Comment:
This should become a "result" too, but we can do that separately: https://github.com/CosmWasm/wasmvm/issues/475